### PR TITLE
hr_homeworking_calendar: fix action path and deletion right

### PR DESCRIPTION
**Pull request by mepe-odoo**
In this task-3642327, the split between
hr_homeworking and calendar was introduced.
The action's path in js was forgotten and the work_location deletion right is set according to the user edit right. 

task-386625

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_homeworking_calendar/static/tests/calendar_homeworking_tests.js [hunk 1]
```diff
@@ -300,14 +310,13 @@ QUnit.module("homeworking", ({ beforeEach }) => {
         assert.deepEqual(worklocationNames, ["Office", "", "", "Home", "Set Location", "Set Location", "Office"]);
 
         await click(workLocations[0], '.o_worklocation_text');
-
```
Comment (Flotchet) [inline addons/hr_homeworking_calendar/static/tests/calendar_homeworking_tests.js:303]: what are thooooooooooooooooooose ?

File: addons/hr_homeworking_calendar/static/tests/calendar_homeworking_tests.js [hunk 2]
```diff
@@ -203,9 +212,10 @@ QUnit.module("homeworking", ({ beforeEach }) => {
                             type: "many2one",
                             relation: "partner",
                         },
+                        can_edit: { string: "User can edit his own record", type: "boolean"},
                     },
                     records: [
-                        { id: uid, display_name: "user 1", partner_id: 1 },
+                        { id: 1, display_name: "user 1", partner_id: 1 , can_edit: true},
```
Comment (Flotchet) [inline addons/hr_homeworking_calendar/static/tests/calendar_homeworking_tests.js:218]: why did u change that? huh?
282fe1d7e40e216c723740b0af255952

**Review by Flotchet — APPROVED** [review]

File: addons/hr_homeworking_calendar/wizard/homework_location_wizard.py [hunk 3]
```diff
@@ -23,6 +23,10 @@ def _compute_day_week_string(self):
         for record in self:
             record.day_week_string = record.date.strftime("%A")
 
+    @api.depends('date')
+    def _compute_user_can_edit(self):
+        self.user_can_edit = self.env.user.has_group("hr.group_hr_user") or self.env.user.can_edit
```
Comment (tivisse) [inline addons/hr_homeworking_calendar/wizard/homework_location_wizard.py:28]:
```suggestion
        self.user_can_edit = self.env.user.can_edit
```
as 
```py
    def _compute_can_edit(self):
        can_edit = self.env['ir.config_parameter'].sudo().get_param('hr.hr_employee_self_edit') or self.env.user.has_group('hr.group_hr_user')
        for user in self:
            user.can_edit = can_edit
```

File: addons/calendar/static/tests/disable_patch.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+/** @odoo-module */
+import {
+        unpatchAttendeeCalendarCommonPopover,
+        unpatchAttendeeCalendarCommonPopoverClass
+    } from "@hr_homeworking_calendar/calendar/common/popover/calendar_common_popover";
+
+unpatchAttendeeCalendarCommonPopover();
+unpatchAttendeeCalendarCommonPopoverClass();
```
[the remark this answers is not in the document]
Comment (mepe-odoo) [reply addons/calendar/static/tests/disable_patch.js:8]: nice catch x) I put it in the wrong module :angel:

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**